app/views/contacts/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="container mt-3" style="padding: 90px">
+  <div class="row justify-content-center">
+    <div class="col-12" %>
+      <h1>Edit <%= @contact.first_name %> <%= @contact.last_name %></h1>
+
+      <%= render 'form', contact: @contact %>
+
```

Comment:
@jackmiddleton always check (or even count) your closing divs, you're miss a few of these here